src/Acuminator/Acuminator.Analyzers/StaticAnalysis/EventHandlers/EventHandlerAnalyzer.cs

```diff
@@ -75,17 +75,17 @@ protected override void AnalyzeSymbol(SymbolAnalysisContext context, PXContext p
 			});
 		}
 
-		private void AnalyzeLambda(OperationAnalysisContext context, PXContext pxContext)
-		{
-			if (context.Operation is ILambdaExpression lambdaExpression)
-			{
-				var symbolAnalysisContext =
-					new SymbolAnalysisContext(lambdaExpression.Signature, context.Compilation, context.Options,
-											  context.ReportDiagnostic, d => true, // this check is covered inside context.ReportDiagnostic
-											  context.CancellationToken);
+		//private void AnalyzeLambda(OperationAnalysisContext context, PXContext pxContext)
```

Comment:
Do we need this? Please delete if not.

Reply:
We already discussed it. We need to investigate more what happened to this API in Roslyn 2.0. I suggest to keep it for now

Reply:
I remember that we discussed it, but it was about merge in the feature branch, not it is dev branch. Do we have an item for this task?

Reply:
I remember that we discussed it, but it was about merge in the feature branch, not it is dev branch. Do we have an item for this task?

Reply:
No, I will create a ticket

Reply:
https://jira.acumatica.com/browse/ATR-397